sdk/src/manifest_store.rs

```diff
@@ -257,7 +257,7 @@ mod tests {
         assert!(manifest_store.validation_status().is_none());
         let manifest = manifest_store.get_active().unwrap();
         assert!(!manifest.ingredients().is_empty());
-        assert_eq!(manifest.issuer().unwrap(), "Some Company");
+        assert_eq!(manifest.issuer().unwrap(), "C2PA Test Signing Cert");
```

Comment:
This change might break some test cases, I'll check. And it will display in all the test images we generate when you view them on verify.

Reply:
Yes it will break this test case:
    fn manifest_report_image() {
        ...
        assert_eq!(manifest.issuer().unwrap(), "Some Company");

Reply:
That is why there are new test images.

Reply:
I am not sure I understand the first comment. "Some Company" would also show on the verify site.  I think the new change is more clear.